tests/aws/services/events/test_events_schedule.py

```diff
@@ -0,0 +1,347 @@
+                .paginate(logGroupName=logs_create_log_group)
+                .build_full_result()
+            )
+            _events.extend(_response["events"])
+
+            if len(_events) < 2:
+                raise AssertionError(
+                    f"Expected at least two events in log group streams, was {_events}"
+                )
+            return _events
+
+        events = retry(_get_events, retries=5)
+
+        events.sort(key=lambda event: event["timestamp"])
+
+        snapshot.match("log-events", events)
+
+
+class TestScheduleCron:
+    @markers.aws.validated
+    @pytest.mark.parametrize(
+        "schedule_cron",
+        [
+            "cron(0 10 * * ? *)",  # Run at 10:00 am every day
+            "cron(15 12 * * ? *)",  # Run at 12:15 pm every day
+            "cron(0 18 ? * MON-FRI *)",  # Run at 6:00 pm every Monday through Friday
+            "cron(0 8 1 * ? *)",  # Run at 8:00 am on the 1st day of every month
+            "cron(0/15 * * * ? *)",  # Run every 15 minutes
+            "cron(0/10 * ? * MON-FRI *)",  # Run every 10 minutes Monday through Friday
+            "cron(0/5 8-17 ? * MON-FRI *)",  # Run every 5 minutes Monday through Friday between 8:00 am and 5:55 pm
+            "cron(0/30 20-23 ? * MON-FRI *)",  # Run every 30 minutes between 8:00 pm and 11:59 pm Monday through Friday
+            "cron(0/30 0-2 ? * MON-FRI *)",  # Run every 30 minutes between 12:00 am and 2:00 am Monday through Friday
+        ],
+    )
+    def tests_put_rule_with_schedule_cron(
+        self, schedule_cron, events_put_rule, aws_client, snapshot
+    ):
+        rule_name = f"rule-{short_uid()}"
+        snapshot.add_transformer(snapshot.transform.regex(rule_name, "<rule-name>"))
+
+        response = events_put_rule(Name=rule_name, ScheduleExpression=schedule_cron)
+        snapshot.match("put-rule", response)
+
+        response = aws_client.events.list_rules(NamePrefix=rule_name)
+        snapshot.match("list-rules", response)
+
+    @markers.aws.validated
+    def test_schedule_cron_target_sqs(
+        self,
+        create_sqs_events_target,
+        events_put_rule,
+        aws_client,
+        snapshot,
+    ):
+        queue_url, queue_arn = create_sqs_events_target()
+
+        schedule_cron, target_datetime = get_cron_expression(
+            1
+        )  # only next full minut might be to fast for setup must be UTC time zone
+
+        bus_name = "default"
+        rule_name = f"test-rule-{short_uid()}"
+        events_put_rule(Name=rule_name, EventBusName=bus_name, ScheduleExpression=schedule_cron)
+
+        target_id = f"target-{short_uid()}"
+        aws_client.events.put_targets(
+            Rule=rule_name,
+            EventBusName=bus_name,
+            Targets=[
+                {"Id": target_id, "Arn": queue_arn},
+            ],
+        )
+
+        time.sleep(120)  # required to wait for time delta 1 minute starting from next full minute
```

Comment:
although cron tab fires in 1 minute, it is required to wait until start of next full minute to begin with due to cold start on AWS.